### PR TITLE
Fix Make a Person instructions

### LINKED ISSUE #7366: Make a Person: no definition of "arity"
It's strange that the description for this exercise says "All functions that take an argument have an **arity **of 1, and the argument will be a string.", when it's simpler just to say something like "All functions that take arguments, take one argument that is a string". Also I feel that the word "arity" is an outdated word after googling the word and finding out that it used to be a property of functions but is now [deprecated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/arity)

### PR DESCRIPTION
**Pull request by eeseesee**
Removed confusing use of the word "arity" from the challenge description and updated the challenge wording to be more straightforward. I did not update the Spanish wording.

closes #7366

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 1]
```diff
@@ -324,9 +324,15 @@
...
-        "All functions that take an argument have an arity of 1, and the argument will be a string.",
+        "Fill in the object constructor with the methods specified below:",
+        "getFirstName()",
+        "getLastName()",
+        "getFullName()",
+        "setFirstName(first)",
+        "setLastName(last)",
+        "setFullName(firstAndLast)",
+        "Run the tests to see the expected output for each method.",
+        "For the functions that take an argument, they should take only one string as an argument.",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: `The functions that take an argument, must accept only one and it has to be a string.`

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 2]
```diff
@@ -324,9 +324,15 @@
       "id": "a2f1d72d9b908d0bd72bb9f6",
       "title": "Make a Person",
```
Comment (justinian336) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Traducción sugerida:

"Completa el constructor de objetos con los métodos especificados a continuación:",
"getFirstName()",
"getLastName()",
"getFullName()",
"setFirstName(first)",
"setLastName(last)",
"setFullName(firstAndLast)",
"Ejecuta las pruebas para ver el resultado esperado de cada método.",
"Las funciones que aceptan argumentos deben aceptar sólo uno, y este tiene que ser una cadena.",
"Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Leer-Buscar-Preguntar</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."

Comment (vtamara) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: @justinian336 suggested a good translation (from it I just translated first line and escaped \<. We will wait few days that @eeseesee updates the pull request adding the translation (by changing in your branch and running <code>git commt -a --amend</code>).

**Comment by SaintPeter** [comment]
@eeseesee Will you be updating this as suggested by @justinian336?

**Comment by SaintPeter** [comment]
Closing as stale.  If you would like to make the changes you may re-submit.

**Pull request closed without merging**